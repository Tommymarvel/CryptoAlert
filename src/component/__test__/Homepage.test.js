import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/configure';

import HomePage from '../Homepage';

describe('HomePage', () => {
  it('renders correctly', () => {
    const tree = Renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <HomePage />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
