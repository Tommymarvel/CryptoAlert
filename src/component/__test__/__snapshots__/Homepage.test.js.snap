// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
Array [
  <h1>
    TOP CURRENCIES
  </h1>,
  <div
    className="list"
  />,
]
`;
